spec/unit/outputs/elasticsearch/http_client_spec.rb

```diff
@@ -97,6 +97,49 @@
     end
   end
 
+  describe "bulk response" do
```

Comment:
I suggest `describe "join_bulk_responses" do` instead, this way we're describing the behaviour of this method.